testutil/util.go

```diff
@@ -64,6 +66,7 @@ func startInProcess(cfg Config, val *Validator) error {
 			WithHomeDir(tmCfg.RootDir).
 			WithChainID(cfg.ChainID).
 			WithJSONMarshaler(cdc).
+			WithTxGenerator(simapp.MakeEncodingConfig().TxGenerator).
```

Comment:
You cannot assume simapp is used here. You should consider cherry-picking my changes in https://github.com/cosmos/cosmos-sdk/pull/6525 for the `testutil` pkg.